bin/main.dart

```diff
@@ -7,27 +7,27 @@ import 'package:localized/constants.dart';
 import 'package:translator/translator.dart';
 import 'package:tuple/tuple.dart';
 
-/// Supported translation providers
-/// GoogleTest should be on the first place
-///
+/// Supported translation providers.
+/// The default provider should be the first
+
+class _Provider {
+  final name;
```

Comment:
It is good when you provide types during declarations, like String, int, etc.

Reply:
Agree